connectors/sqs/src/test/java/io/camunda/connector/inbound/SqsExecutableTest.java

```diff
@@ -0,0 +1,166 @@
+import io.camunda.connector.common.suppliers.SqsGsonComponentSupplier;
+import io.camunda.connector.impl.inbound.InboundConnectorProperties;
+import io.camunda.connector.impl.inbound.correlation.StartEventCorrelationPoint;
+import io.camunda.connector.impl.inbound.result.MessageCorrelationResult;
+import io.camunda.connector.inbound.model.SqsInboundProperties;
+import io.camunda.connector.test.inbound.InboundConnectorContextBuilder;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class SqsExecutableTest {
+  private static final String ACTUAL_QUEUE_URL = "https://sqs.region.amazonaws.com/camunda-test";
+  private static final String ACTUAL_QUEUE_REGION = "us-east-1";
+  private static final String AWS_ACCESS_KEY = "AWS_ACCESS_KEY";
+  private static final String AWS_SECRET_KEY = "AWS_SECRET_KEY";
+  private static final String SQS_QUEUE_URL = "SQS_QUEUE_URL";
+  private static final String ACTUAL_ACCESS_KEY = "4W553CR3TK3Y";
+  private static final String ACTUAL_SECRET_KEY = "AAAABBBBCCCDDD";
+  private static final String ATTRIBUTE_NAME = "ATTRIBUTE_NAME_KEY";
+  private static final String ACTUAL_ATTRIBUTE_NAME = "attribute";
+  private static final String MESSAGE_ATTRIBUTE_NAME = "MESSAGE_ATTRIBUTE_NAME_KEY";
+  private static final String ACTUAL_MESSAGE_ATTRIBUTE_NAME = "message attribute";
+
+  private static final Gson GSON = SqsGsonComponentSupplier.gsonInstance();
+  private static final String SUCCESS_CASES_RESOURCE_PATH =
+      "src/test/resources/requests/inbound/success-test-cases.json";
+
+  @Mock private AmazonSQS sqsClient;
+  @Mock private AmazonSQSClientSupplier supplier;
+  private ExecutorService executorService;
+  private AtomicBoolean isQueueConsumerActive;
+
+  @BeforeEach
+  public void setUp() {
+    when(supplier.sqsClient(ACTUAL_ACCESS_KEY, ACTUAL_SECRET_KEY, ACTUAL_QUEUE_REGION))
+        .thenReturn(sqsClient);
+
+    executorService = Executors.newSingleThreadExecutor();
+
+    isQueueConsumerActive = new AtomicBoolean(true);
+  }
+
+  @ParameterizedTest
+  @MethodSource("successRequestCases")
+  @MockitoSettings(strictness = Strictness.LENIENT)
```

Comment:
Why do we need lenient strictness level here? Is it because of Unnecessary Stubbing exception? If so, can't we resolve that, instead of bypassing it?